fuse/moosefs/Portfile

```diff
@@ -0,0 +1,103 @@
+PortSystem		1.0
+
+name			moosefs
+version			3.0.107
+distname		${name}-${version}-1
+categories		fuse
+platforms		darwin
+license			GPL-2
+maintainers		{@chogata moosefs.com:chogata}
```

Comment:
Normally we suggest that `openmaintainer` be added unless there's good reason not to have it.